numba/tests/test_array_methods.py

```diff
@@ -779,18 +787,198 @@ def test_sum(self):
         # OK
         self.assertPreciseEqual(pyfunc(a, 0), cfunc(a, 0))
 
-    def test_sum_kws(self):
-        pyfunc = array_sum_kws
+    def test_sum2(self):
```

Comment:
It's be really helpful for future refactoring if the intent of the test were described as a docstring/comment. Some of the new tests have this and they've been very helpful.

Reply:
good idea, done